### DISCUSSION TITLE
让女仆能够挖矿或者砍树

### DISCUSSION THREAD

**Discussion by 3w3wcom** [TartaricAcid/TouhouLittleMaid#563] [Suggestions & Ideas]
希望女仆能在挖矿或者砍树上面一起进行一些帮助。

可以按照副手或者背包里第1栏的物品而寻找并挖掘一些指定矿物。（或者饰品）
挖矿机制可以格局视野内的距离和矿物种类独立设置一些数值来指定优先级。

虽然说有些不太忍心让女仆一样变成脏兮兮的矿工（，但是还是想要下矿的时候能够有个人一起帮忙_(´ཀ`」 ∠)__

**Comment by TartaricAcid** [comment]
这个目前还在计划中
